containers/unit_tests/TestDualView.hpp

```diff
@@ -520,58 +520,26 @@ namespace {
  * that we keep the semantics of UVM DualViews intact.
  */
 // modify if we have other UVM enabled backends
-#if defined(KOKKOS_ENABLE_CUDA) || defined(KOKKOS_ENABLE_SYCL) || \
-    defined(KOKKOS_ENABLE_HIP)  // OR other UVM builds
-#define UVM_ENABLED_BUILD
-#endif
-
-#ifdef UVM_ENABLED_BUILD
-template <typename ExecSpace>
-struct UVMSpaceFor;
-#endif
-
-#ifdef KOKKOS_ENABLE_CUDA  // specific to CUDA
-template <>
-struct UVMSpaceFor<Kokkos::Cuda> {
-  using type = Kokkos::CudaUVMSpace;
-};
-#endif
 
-#ifdef KOKKOS_ENABLE_SYCL  // specific to SYCL
-template <>
-struct UVMSpaceFor<Kokkos::SYCL> {
-  using type = Kokkos::SYCLSharedUSMSpace;
-};
-#endif
-
-#ifdef KOKKOS_ENABLE_HIP  // specific to HIP
-template <>
-struct UVMSpaceFor<Kokkos::HIP> {
-  using type = Kokkos::HIPManagedSpace;
-};
-#endif
-
-#ifdef UVM_ENABLED_BUILD
-template <>
-struct UVMSpaceFor<Kokkos::DefaultHostExecutionSpace> {
-  using type = typename UVMSpaceFor<Kokkos::DefaultExecutionSpace>::type;
-};
+#ifdef KOKKOS_HAS_SHARED_SPACE
+template <typename ExecutionSpace>
+using SharedTestSpace = Kokkos::SharedSpace;
 #else
-template <typename ExecSpace>
-struct UVMSpaceFor {
-  using type = typename ExecSpace::memory_space;
-};
+template <typename ExecutionSpace>
+using SharedTestSpace = typename ExecutionSpace::memory_space;
```

Comment:
I don't really like that name.  What about `TestSharedSpace`?
Or do you have any other suggestion where we don't interleave a word in the alias name?